e2e/transform-linked-modules/package.json

```diff
@@ -4,7 +4,8 @@
     "transformIgnorePatterns": [
       "/node_modules/",
       "<rootDir>/__tests__",
-      "<rootDir>/ignored/"
+      "<rootDir>/ignored/",
+      "/jest-environment-node/"
```

Comment:
I had to ignore the environment from the transform patterns here as it was completely overriding the environment's code

Reply:
Not sure I understand how exactly this broke, mind elaborating? Just to make sure this can't happen in a real project using Jest

Reply:
The preprocessor used in this test removes the custom environment, and replaces it with it's own export

https://github.com/facebook/jest/blob/5177236f31c705f1084822a9bb7b7da976a420f6/e2e/transform-linked-modules/preprocessor.js#L4-L7

Reply:
In this test the env was `node` but it was taken as a relative path, and not from `node_modules`.

Reply:
Oh I see, sorry for the dumb question ^^